### ISSUE TITLE
Respect priority for unforced break rules

### ISSUE THREAD

**Issue by kwkbtr** [vivliostyle/vivliostyle.js#27]
Spec: [CSS Fragmentation - 4.4. Unforced Breaks](http://dev.w3.org/csswg/css-break/#unforced-breaks)

> If the above doesn’t provide enough break points to keep content from overflowing the fragmentainer, then rules 1, 2 and 4 are dropped in order to find additional breakpoints. In this case the UA may use the avoids that are in effect at those points to weigh the appropriateness of the new breakpoints; however, this specification does not suggest a precise algorithm.

> If that still does not lead to sufficient break points, rule 3 is dropped as well, to find still more break points.

**Comment by frivoal** [comment]
Check the spec, and if updates discussed in Paris have been applied.

**Comment by frivoal** [comment]
Yes, the resolution I was thinking about has been applied (it's commit https://github.com/w3c/csswg-drafts/commit/7b5d13351c9408f587ecf86ddcab714d043c4fa8). This means the quote from the spec above is no longer correct.

The new correct is:

>  If the above doesn’t provide enough break points to keep content from overflowing the fragmentainer, then rule 3 is dropped to provide more break points.
> 
> If that still does not lead to sufficient break points, then rules 1, 2 and 4 are dropped in order to find additional breakpoints. In this case the UA may use the avoids that are in effect at those points to weigh the appropriateness of the new breakpoints; however, this specification does not suggest a precise algorithm.
> 
> If even that does not lead to sufficient break points, cloned margins/border/padding at on the block-end side are truncated; and if more room is still needed, cloned margins/border/padding are truncated at the block-end side as well.
> 
> Finally, if there are no possible break points below the top of the fragmentainer, and not all the content fits, the UA may break anywhere [...]
